### DISCUSSION TITLE
Casual to be renamed to Casual Calc

### DISCUSSION THREAD

**Discussion by kickingvegas** [kickingvegas/casual-calc#153] [Announcements]
A plan is in effect to rename Casual to Casual Calc. More details forthcoming.

**Comment by kickingvegas** [comment]
Details on the renaming plan for Casual to Casual Calc.

http://yummymelon.com/devnull/announcing-a-plan-to-change-some-casual-package-names.html

**Comment by tusharhero** [comment]
I think this is a good decision.

**Comment by kickingvegas** [comment]
Detailed migration plans for Casual Calc #154 and Casual I-Search https://github.com/kickingvegas/cc-isearch-menu/issues/34

**Comment by kickingvegas** [comment]
Nice! Irreal post on this name change. https://irreal.org/blog/?p=12241
